### DISCUSSION TITLE
Hoarder Alfredworkflow

### DISCUSSION THREAD

**Discussion by yinan-c** [karakeep-app/karakeep#677] [Show and tell]
For Hoarder and Alfred users:
https://github.com/yinan-c/alfred-hoarder
![CleanShot 2024-11-19 at 20 29 16@2x](https://github.com/user-attachments/assets/8aa2db72-586c-446c-b527-cda167f0fc1e)

Feel free to leave any suggestions.

**Comment by MohamedBassem** [comment]
That's pretty cool. Thanks for creating it! I'll feature it in the docs :)

**Reply by yinan-c** [reply]
Thank you!
